buildenv/jenkins/JenkinsfileBase

```diff
@@ -144,8 +144,21 @@ def test() {
 		}
 		stage('Test') {
 			timestamps{
-				echo 'Running tests...'
-				sh "$OPENJDK_TEST/maketest.sh $OPENJDK_TEST _$TARGET"
+				if (env.BUILD_LIST == "jck" && env.SPEC == "linux_x86-64"){
+					wrap([$class: 'Xvfb', autoDisplayName: true]) {
+						DISPLAY = sh (
```

Comment:
Using def DISPLAY = 
otherwise it will be a global variable.

Reply:
updated, thanks 👍